src/main/java/tuteez/ui/DisplayCard.java

```diff
@@ -50,23 +53,60 @@ public DisplayCard(Optional<Person> lastViewedPerson) {
         if (lastViewedPerson.isPresent()) {
             name.setText(person.getName().fullName);
             phone.setText(person.getPhone().value);
-            telegram.setText(person.getTelegramUsername().telegramUsername);
+            setTelegramUsernameText(person);
             address.setText(person.getAddress().value);
             email.setText(person.getEmail().value);
-
-            tags.getChildren().clear();
             person.getTags().stream()
                     .sorted(Comparator.comparing(tag -> tag.tagName))
-                    .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
-
-            remarks.getChildren().clear();
-            person.getRemarkList().getRemarks().stream()
-                    .sorted(Comparator.comparing(remark -> remark.toString()))
-                    .forEach(remark -> remarks.getChildren().add(new Label(remark.toString())));
+                    .forEach(tag -> displayTags.getChildren().add(new Label(tag.tagName)));
+            IntStream.range(0, person.getRemarkList().getRemarks().size())
+                    .forEach(i -> {
+                        String remark = person.getRemarkList().getRemarks().get(i).toString();
```

Comment:
Perhaps it is better to functionise/abstract this? Chaining too many getters may make code less safe and hard to maintain